src/Motor.Extensions.Hosting/Internal/BackgroundTaskQueue.cs

```diff
@@ -20,6 +20,7 @@ public BackgroundTaskQueue(IMetricsFactory<BackgroundTaskQueue<T>>? metricsFacto
         _elementsInQueue = metricsFactory?.CreateGauge("task_queue_enqueued_elements", "", false, "type")
             ?.WithLabels(typeof(T).Name);
         _totalMessages = metricsFactory?.CreateCounter("total_messages", "", false, "type")?.WithLabels(typeof(T).Name);
+        LastDequeuedAt = DateTimeOffset.UtcNow;
```

Comment:
nit: I'd prefer to rather have this as the default value in [line 59](https://github.com/GDATASoftwareAG/motornet/pull/974/files#diff-2aad6d2e06e8eb351c0feba7943f1557852c0662babf0050f23148c632c316a1R59) instead of here as a new line in the constructor.